app/assets/stylesheets/minister-select.scss

```diff
@@ -1,36 +1,28 @@
 @import 'colours';
-
+// Colours for the site, as per:
+// http://alphagov.github.com/design/gov.uk.colours/
 input.minister-select-input {
-  width: 248px;
+    width: 248px;
 }
-
 .autocomplete-suggestions {
-  border: 1px solid $grey-2;
-  background: $white;
-  overflow: auto;
+    border: 1px solid $grey-2;
+    background: $white;
+    padding: 2px 5px;
+    white-space: nowrap;
+    overflow: hidden;
 }
-
-.autocomplete-suggestion {
-  padding: 2px 5px;
```

Comment:
I think you've merged autocomplete-suggestion (no trailing s) with autocomplete-suggestions. Didn't it show in the app?